app/src/main/java/org/simple/clinic/editpatient/EditPatientModel.kt

```diff
@@ -29,7 +30,7 @@ data class EditPatientModel(
         phoneNumber: PatientPhoneNumber?,
         dateOfBirthFormatter: DateTimeFormatter,
         bangladeshNationalId: BusinessId?,
-        saveButtonState: EditPatientState?
+        saveButtonState: EditPatientState?,
```

Comment:
Not necessary to remove it in this PR. But seems like there is an extra comma here.